macros/nuPRISMBeamTest.mac

```diff
@@ -0,0 +1,164 @@
+# nuPRISMBeamTest WCTE macro with no visualization
+
+/run/verbose 1
+/tracking/verbose 0
+/hits/verbose 0
+#/control/execute vis.mac
+#/vis/open OGLSQt
+
+## select the geometry
+# Default config if you do nothing is currently SuperK
+# Nominal geometry for WCTE is nuPRISMBeamTest_16cShort_mPMT which is uncommented
+
+#/WCSim/WCgeom nuPRISM
+
+#Select which PMT to use
+#/WCSim/nuPRISM/SetPMTType PMT8inch
+#/WCSim/nuPRISM/SetPMTPercentCoverage 40
+#Set height of nuPRISM inner detector
+#/WCSim/nuPRISM/SetDetectorHeight 6. m
+#Set vertical position of inner detector, in beam coordinates
+#/WCSim/nuPRISM/SetDetectorVerticalPosition 0. m
+#Set diameter of inner detector
+#/WCSim/nuPRISM/SetDetectorDiameter 8. m
+# Set Gadolinium doping (concentration is in percent)
+#/WCSim/DopingConcentration 0.1
+#/WCSim/DopedWater false
+#/WCSim/Construct
+
+#Use mPMTs settings (uncomment/delete the above)
+#/WCSim/WCgeom nuPRISM_mPMT
+#/WCSim/WCgeom nuPRISMBeamTest_mPMT ## this is 18c5r from the original design
+/WCSim/WCgeom nuPRISMBeamTest_16cShort_mPMT ## this is 16c4r from CAD
+#/WCSim/WCgeom nuPRISMShort_mPMT
+# Set Gadolinium doping (concentration is in percent)
+#/WCSim/DopingConcentration 0.1
+#/WCSim/DopedWater false
+/WCSim/Construct
+## OR for single mPMT mode or updating mPMT parameters:
+#/control/execute macros/mPMT_nuPrism1.mac         ## mPMT options: mPMT_nuPrism1.mac and 2.mac
+
+#Added for the PMT QE option 08/17/10 (XQ)
+# 1. Stacking only mean when the photon is generated
+# the QE is applied to reduce the total number of photons
+# 2. Stacking and sensitivity detector
+# In the stacking part, the maximum QE is applied to reduce
+# the total number of photons
+# On the detector side, the rest of QE are applied according to QE/QE_max
+# distribution. This option is in particular important for the WLS
+# 3. The third option means all the QE are applied at the detector
+# Good for the low energy running.
+# 4. Switch off the QE, ie. set it at 100%
+
+/WCSim/PMTQEMethod     Stacking_Only
+#/WCSim/PMTQEMethod     Stacking_And_SensitiveDetector
+#/WCSim/PMTQEMethod     SensitiveDetector_Only
+#/WCSim/PMTQEMethod     DoNotApplyQE
+
+#turn on or off the collection efficiency
+/WCSim/PMTCollEff on
+
+# command to choose save or not save the pi0 info 07/03/10 (XQ)
+/WCSim/SavePi0 false
+
+#choose the Trigger & Digitizer type (and options)
+/DAQ/Digitizer SKI
+/DAQ/Trigger NDigits
+
+#grab the other DAQ options (thresholds, timing windows, etc.)
+/control/execute macros/daq.mac
+
+# setseed - option to change seeds
+#/random/setSeeds 2 1
+#/WCSim/random/seed 4
+
+
+# default dark noise frequency (and conversion factor) is PMT property (NEW), set in the code.
```

Comment:
again, this whole DR section should go in mPMT macro? and also remove all the SKDETSIM/20" PMT options?